modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -128,9 +128,17 @@ include::howtos:example$provisioning_resources_collections.py[tag=drop_collectio
 include::howtos:example$provisioning_resources_collections.py[tag=drop_scope]
 ----
 
-Note that you require at least the https://docs.couchbase.com/server/7.0/learn/security/roles.html#bucket-admin[Bucket Admin] privilege to create or drop a Scope.
-The same role also lets you manage Collections, though a more minimal set of privileges is https://docs.couchbase.com/server/7.0/learn/security/roles.html#manage-collections-in-scope[Manage Collections in Scope] along with https://docs.couchbase.com/server/7.0/learn/security/roles.html#data-reader[Data Reader].
+Note that the most minimal permissions to create and drop a Scope or Collections is
```

Comment:
Should be `Collection` here instead of `Collections`.

Reply:
Oops,  ta!